README.md

```diff
@@ -126,6 +126,15 @@ params={{
       }}
 ```
 
+##### `marginBottom`
+
+The `marginBottom` props specifies a custom value for `margin-bottom` CSS property for `.conversation-container` class, this changes how much the container will appear above the close button.
+```css
+marginBottom: `.conversation-container {
```

Comment:
In this case, can I configure other CSS properties? If yes, I would suggest to change the name from `marginBottom` to `customCss`

Reply:
this could already be done with the `defaultHighlightAnimation` property, the string could have any kind of css properties on it.

Reply:
hmm, so both work in the same way? maybe this new one is just a more suggestive property name to do so. what do u think?

Reply:
We can use this `defaultHighlightAnimation` with a different name, and use just one.

Reply:
If we change this property name, all current projects will not be able to customize this property, and this will lead to issues until they change on their projects the property name.

I think the best path to choose is, change this new property name to `customCss`, like you said, deprecate the `defaultHighlightAnimation` property and remove it in future versions. So for now, current projects will not break and can migrate to the newer version without problems, and new projects will know that `defaultHighlightAnimation` should not be used anymore.